src/components/Person/Person.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="person">
+    <h2 className="person__name">{`My name is ${name}`}</h2>
+    <p className="person__age">
+      {age && `I am ${age}`}
+    </p>
```

Comment:
```suggestion
    {age && (
      <p className="person__age">
        {`I am ${age}`}
      </p>
```
Зараз ти рендериш пустий параграф, якщо немає age. Давай взагалі не рендерити у такому випадку

Reply:
Да, это выглядит логичным, спасибо) исправил.